tests/vmi_multus_test.go

```diff
@@ -665,6 +666,28 @@ var _ = Describe("SRIOV", func() {
 			return
 		}
 
+		createSriovVMI := func(networksNames []string) (vmi *v1.VirtualMachineInstance) {
+			SriovVMI := getSriovVmi(networksNames)
+
+			// Explicitly choose different random mac addresses instead of relying on kubemacpool to do it:
+			// 1) we don't at the moment deploy kubemacpool in kind providers
+			// 2) even if we would do, it's probably a good idea to have the suite not depend on this fact
+			//
+			// This step is needed to guarantee that no VFs on the PF carry a duplicate MAC address that may affect
+			// ability of VMIs to send and receive ICMP packets on their ports.
+			for _, iface := range SriovVMI.Spec.Domain.Devices.Interfaces {
+				if iface.InterfaceBindingMethod.SRIOV != nil {
+					mac, err := tests.GenerateRandomMac()
+					Expect(err).ToNot(HaveOccurred(), "mac address should be correctly generated: %v", err)
+					SriovVMI.Spec.Domain.Devices.Interfaces[1].MacAddress = mac.String()
```

Comment:
Why are you iterating over all the interface to always the set mac on interface 1?

Reply:
my mistake. should be set on any SRIOV interface. 
Fixed.